hooks/cron.ts

```diff
@@ -0,0 +1,43 @@
+import { ofetch, $Fetch } from "ofetch";
+import { CronJob } from "cron";
+import { publicRuntimeConfig } from "../nuxt.config";
+import { getNextQueuedScene, popNextQueuedScene } from "../utils/apis";
+
+async function dailyFeatureSetup(backendCall: $Fetch) {
+  const now = new Date();
+  const tomorrow = new Date(now);
+  tomorrow.setDate(tomorrow.getDate() + 1);
+  const features = await getFeaturesInRange(backendCall, now.getTime(), tomorrow.getTime());
+
+  if (features.length === 0) {
+    features.forEach(feature => {
+      const date = new Date(feature.feature_time);
+      const featureJob = new CronJob(date, () => {
```

Comment:
I think that one can get away without this, but I think it would be better to mark this closure as `async` and `await` the result of `miscUpdateTimeline`. I think that it would be equivalent to return that the update function's return value, which will be a Promise, but I like having the async-ness more explicit.